WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -417,3 +439,83 @@ extension OrderDetailsViewModel {
         stores.dispatch(deleteTrackingAction)
     }
 }
+
+extension OrderDetailsViewModel: OrderDetailsViewModelStatusSubModel {
+
+    // TODO
+    // configureResultsController
+    //X statusResultsController.startForwardingEvents(to: tableView)
+
+    func refreshStatuses() {
+        do {
+            try statusResultsController.performFetch()
+        } catch {
+            CrashLogging.logError(error)
+        }
+    }
+
+    func getNumStatusSections() -> Int {
+        return statusResultsController.sections.count
+    }
+
+    func getNumStatuses(inSection section: Int) -> Int {
+        return statusResultsController.sections[section].numberOfObjects
+    }
+
+    func getStatusName(at indexPath: IndexPath) -> String? {
+        let status = statusResultsController.object(at: indexPath)
```

Comment:
Swift method names, like Obj-C, also typically don't have `get` in them. Something like this can be:

```swift
func statusName(at indexPath: IndexPath) -> String? {
```

It's weird, I know. 😄 But following that convention makes it easier for everyone else to read I think.

Reply:
Fixed "get" in https://github.com/woocommerce/woocommerce-ios/pull/3530/files#diff-931cf23b4f72cbc59934f4caaffdc26bb12776a303513ead86d06d0e8ff69914R59

Fixed one-liner in https://github.com/woocommerce/woocommerce-ios/pull/3530/files#diff-931cf23b4f72cbc59934f4caaffdc26bb12776a303513ead86d06d0e8ff69914R26